xbmc/CompileInfo.cpp.in

```diff
@@ -60,13 +60,13 @@ const char* CCompileInfo::GetCopyrightYears()
   return "@APP_COPYRIGHT_YEARS@";
 }
 
-const char* CCompileInfo::GetBuildDate()
+const std::string CCompileInfo::GetBuildDate()
```

Comment:
Either pass by const reference or by non-const value.
I don't see any benefit for passing by const value.

Reply:
update to pass by non-const value